requirements.txt

```diff
@@ -1,8 +1,10 @@
 codecov
 coverage
 flake8
+h5py
```

Comment:
Ah, I remember now why I didn't include h5py and numpy in the requirements.txt --- it caused issues on readthedocs.

I think we'll need a few different requirements.txt files, hopefully being able to 'source' each other including more-and-more specific requirements.